test/Microsoft.SqlTools.Dmp.Hosting.UnitTests/ExtensibilityTests/ExtensionTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Composition;
+using System.Linq;
+using System.Reflection;
+using Microsoft.SqlTools.Dmp.Hosting.Extensibility;
+using Xunit;
+
+namespace Microsoft.SqlTools.Dmp.Hosting.UnitTests.ExtensibilityTests
+{
+    public class ExtensionTests
+    {
+
+//        [Fact]
```

Comment:
Were these always commented out? If so, we can remove the class entirely since it has no value.

Reply:
I just verified these aren't commented in the existing code. Please create an issue for this and we need confirmation you'll fix these as part of the remaining work to get this rearchitecture done.